### PR TITLE
Basic pytest integration

### PR DESCRIPTION
**Pull request by priv-kweihmann**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,6 +11,10 @@
 with open('requirements.txt') as f:
     requirements = f.read().splitlines()
 
+requirements_dev = []
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
```
Comment (jens-erdmann) [inline setup.py]: I suppose this was meant to read `requirements_dev`?

File: setup.py [hunk 2]
```diff
@@ -11,6 +11,10 @@
 with open('requirements.txt') as f:
     requirements = f.read().splitlines()
 
+requirements_dev = []
+with open('requirements.txt') as f:
```
Comment (jens-erdmann) [inline setup.py]: And this was meant to read the file `requirements_dev.txt` correct?

File: setup.py (hunk 1 above)
Comment (priv-kweihmann) [reply setup.py]: I have literally no idea what you are talking about :-)

File: setup.py (hunk 2 above)
Comment (priv-kweihmann) [reply setup.py]: same as above - just fixed it in the latest commit

File: tests/test_compat_matrix.py [hunk 3]
```diff
@@ -16,32 +16,23 @@
...
+        self.assertEquals(_compat_matrix.a_compatible_with_b("BSD-3-Clause", "MIT"), CompatMatrixStatus.True)
         # BSD is not a license, should give a None
         print("Below test will output (stderr) a message that compatibility could not be checked", file=sys.stderr)
-        self.assertTrue(compat_matrix.a_compatible_with_b("MIT", "BSD")==None)
+        self.assertRaises(Exception, _compat_matrix.a_compatible_with_b, "MIT", "BSD")
         # GPL-2.0 can use BSD-3-Clause - but not the other way around
-        self.assertTrue(compat_matrix.a_compatible_with_b("GPL-2.0-only", "BSD-3-Clause")==True)
-        self.assertTrue(compat_matrix.a_compatible_with_b("BSD-3-Clause", "GPL-2.0-only")==False)
+        self.assertEquals(_compat_matrix.a_compatible_with_b("GPL-2.0-only", "BSD-3-Clause"), CompatMatrixStatus.True)
+        self.assertEquals(_compat_matrix.a_compatible_with_b("BSD-3-Clause", "GPL-2.0-only"), CompatMatrixStatus.FALSE)
```
Comment (jens-erdmann) [inline tests/test_compat_matrix.py:35]: Should this read `CompatMatrixStatus.False` instead?

**Review by jens-erdmann — CHANGES_REQUESTED** [review]
The rest looks fine to me.

File: tests/test_compat_matrix.py (hunk 3 above)
Comment (priv-kweihmann) [reply tests/test_compat_matrix.py:35]: Nope... https://github.com/vinland-technology/flict/blob/76f8b9cac5a4fc3a16862e20f49594f98d7fdf63/flict/flictlib/compat_matrix.py#L29

But the line above should be `CompatMatrixStatus.TRUE` instead

Comment (jens-erdmann) [reply tests/test_compat_matrix.py:35]: Ha ok. But you fixed the inconsistency.

**Review by jens-erdmann — APPROVED** [review]
LGTM

File: tests/test_compat_matrix.py (hunk 3 above)
Comment (priv-kweihmann) [reply tests/test_compat_matrix.py:35]: Yes just did - interestingly the test succeeds either way

**Pull request merged**